specification.md

```diff
@@ -223,8 +223,8 @@ The workflow data output is the data output of the last executed workflow state.
 
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
-| input | Workflow expression to filter the states data input | string | no |
-| output | Workflow expression that filters the states data output | string | no |
+| input | Workflow expression to filter the states data input | expression | no |
```

Comment:
the type should be a valid JSON type. not a made up one please

Reply:
same for all other changes from "string" type to "expression" type

Reply:
a link is a good idea, thx!